week-1/Homework/F-strings-methods/exercise2.js

```diff
@@ -1,3 +1,4 @@
-const name = " Daniel  ";
+const name = " Burçak  ";
+const nameLength = name.trim().length;
 
-console.log(message);
+console.log('My name is ' + name.trim() + ' and my name is ' + nameLength + ' characters long.');
```

Comment:
You could optimize a bit your code by storing the result of `name.trim()` in an intermediate variable to avoid calling trim twice. For the sake of this example, let's say that trim usually takes 1s to return the trimmed value. Therefore, calling it a 2nd time when not actually needed delays your function by 1 extra sec.

```
const name = " Burçak  ";
const trimmedName = name.trim();
const nameLength = trimmedName.length;

// at this point, trimmedName is available to be used instead of name.trim()
```